content/ubuntu-os.md

```diff
@@ -0,0 +1,30 @@
+# Stemcell OS: Ubuntu
+
+The base Ubuntu operating system is built from the [cloudfoundry/bosh-linux-stemcell-builder](https://github.com/cloudfoundry/bosh-linux-stemcell-builder) repository.
+
+The most recent versions of these distributions are built every other week to ensure the latest upstream patches are included. Critical and High CVEs result in new builds, regardless of the biweekly interval. These builds are published on [bosh.io](https://bosh.io/) as new, minor versions.
```

Comment:
Maybe that's just me as a non-native speaker: "every other week" is less clear than "every two weeks". Can we change that? Or maybe that's just me?

Reply:
Improved; thanks